### PR TITLE
Remove glam dependency and bump version to 0.3.0

### PR DESCRIPTION
**Pull request by bonsairobo**
There was no reason to use glam and it just makes it harder to maintain this crate.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ pub struct Frame<T> {
 }
 
 impl<T: Copy> Frame<T> {
-    pub fn frame_as_vec3a(&self) -> impl Iterator<Item = Vec3> + '_
+    pub fn frame_as_vec3a(&self) -> impl Iterator<Item = [f32; 3]> + '_
```
Comment (plof27) [inline src/lib.rs:29]: This should probably be renamed for clarity, since it's not actually a Vec3A anymore

**Review by plof27 — APPROVED** [review]
One small nitpick, but not enough for me to block a merge. Looks good!

File: src/lib.rs (hunk 1 above)
Comment (bonsairobo) [reply src/lib.rs:29]: Yea I also noticed this and I'm working on it now.

Comment (bonsairobo) [reply src/lib.rs]: I just made it `impl IntoIterator` instead.

**Pull request merged**